### ISSUE TITLE
[Bug]: Desync kick with new FAL

### ISSUE THREAD

**Issue by creenl1** [Mr-X-GTA/YimMenu#67]
### Describe the bug

I get a desync kick. FSL was just approved in unonowncheats. The bypass may not work

### Steps To Reproduce

-

### Expected Behavior

I get a desync kick in public lobby

### Logs

I dont look

### Store

Rockstar

### Language

English

### YimMenu Language

English (US)

### Build

- [X] I've confirmed the issue exists on the latest version of YimMenu

### Coload

- [X] I've tested this without co-loading anything.

**Comment by Lochos97** [comment]
True, even with modder as a host, i keep gettin a weird desync kick, louadin in clouds and the rejoinin the same session every 2-3 mins again, again and agan. Hopefully theres a way to fix this.

**Comment by creenl1** [comment]
I thought you had a bypass that kicks the host until I get the host

**Comment by creenl1** [comment]
I think that in the last updates of YimMenu the host was always kicked or a message was sent to Battleeye that the host was kicked. It's a shame, it's not really enjoyable.

**Comment by AncientTenshi** [comment]
As a hoster, my friend also has the same issue he comes in, he gets kick with message of being de-synced as well.
i dont have any logs to even check if its the mod or Battle-eye because it creates no log system like before (things are broken,i know update takes time). He's using the same mod/scripts/FLS as me. Stand users that not injected seem to come in my session with no issues though.

**Comment by alwinlubbers** [comment]
It looks like the desync issue hasn't been fixed. I'm still getting kicked after 15-20 minutes in a public session (no host).
